### PR TITLE
Add DALL-E mini support for `Image Prompt` mode

### PR DESCRIPTION
**Pull request by dilithjay**
This PR enables the `mini` version of the [DALL-E mini](https://huggingface.co/dalle-mini/dalle-mini) model available here. I will add the additional setting-up steps to the Wiki, once the PR is merged.

Instructions for setting up the files for DALL-E mini:
```
cd models
git clone https://huggingface.co/dalle-mini/dalle-mini
cd dalle-mini
git lfs pull

cd ..
git clone https://huggingface.co/dalle-mini/vqgan_imagenet_f16_16384
cd vqgan_imagenet_f16_16384
git lfs pull
```

The left side panel now looks as follows. The default option for `Model` is `Stable Diffusion` which includes the previously available radio buttons.
image

In addition to the `seed` the following settings are supported:
image

### CODE REVIEW

File: img_styler/image_prompt/dalle_mini_model.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+from PIL import Image
+from dalle_mini import DalleBart, DalleBartProcessor
+from vqgan_jax.modeling_flax_vqgan import VQModel
+
+# Load the config file to read in system settings.
+base_path = (Path(__file__).parent / "../configs/").resolve()
+app_settings = toml.load(f"{base_path}/settings.toml")
+
+
+class DalleMini:
```
Comment (pramitchoudhary) [inline img_styler/image_prompt/dalle_mini_model.py:20]: Was able to get it working. Tried re-producing few concepts from --> https://openai.com/blog/dall-e/
Screen Shot 2022-12-14 at 9 41 06 AM

**Observation/Questions:**
- Call to `p_generate` seemed slow. It could be the hardware I m using, though. Wondering if there are tricks to make it faster?
- If we swap DALLE-mini weights with `-mega` n `settings.toml` update then things should work, right? 
- Also, throws an exception if seed is `empty` default `None`

**Review by pramitchoudhary — APPROVED** [review]
Thanks, @dilithjay, let merge it n then we can make improvements.
Noticed one exception with seed, captured in previous comment.

File: img_styler/image_prompt/dalle_mini_model.py (hunk 1 above)
Comment (dilithjay) [reply img_styler/image_prompt/dalle_mini_model.py:20]: * It was quite slow on my end as well, but it was faster than the case with stable diffusion. Any chance it is not using the GPU? I think it would give a warning if that's the case, but it doesn't error out. There were several steps to install JAX for GPU, which I hadn't mentioned in the description.
* Yes, I think it should. The mega model was too large for my memory so I wasn't able to properly test it out.
* Ah, will fix it before merging.
